src/dotnet/commands/dotnet-list/dotnet-list-package/ListPackageReferencesCommand.cs

```diff
@@ -0,0 +1,197 @@
+        {
+            var result = NuGetCommand.Run(TransformArgs());
+
+            return result;
+        }
+
+        private string[] TransformArgs()
+        {
+            var args = new List<string>
+            {
+                "package",
+                "list",
+            };
+
+            args.Add(GetProjectOrSolution());
+
+            if (_appliedCommand.HasOption("outdated"))
+            {
+                args.Add("--outdated");
+            }
+
+            if (_appliedCommand.HasOption("include-transitive"))
+            {
+                args.Add("--include-transitive");
+            }
+
+            if (_appliedCommand.HasOption("framework"))
+            {
+                //Forward framework as multiple flags
+                foreach (var framework in _appliedCommand.AppliedOptions["framework"].Arguments)
+                {
+                    args.Add("--framework");
+                    args.Add(framework);
+                }
+            }
+
+            if (_appliedCommand.HasOption("include-prerelease"))
+            {
+                CheckForOutdated("--include-prerelease");
+                args.Add("--include-prerelease");
+            }
+
+            if (_appliedCommand.HasOption("highest-patch"))
+            {
+                CheckForOutdated("--highest-patch");
+                args.Add("--highest-patch");
+            }
+
+            if (_appliedCommand.HasOption("highest-minor"))
+            {
+                CheckForOutdated("--highest-minor");
+                args.Add("--highest-minor");
+            }
+
+            if (_appliedCommand.HasOption("config"))
+            {
+                CheckForOutdated("--config");
+                args.Add("--config");
+                //Config path absolute path
+                var configPath = PathUtility.GetAbsolutePath(PathUtility.EnsureTrailingSlash(Directory.GetCurrentDirectory()),
+                                        _appliedCommand.AppliedOptions["config"].Arguments.Single());
+                args.Add(configPath);
+            }
+
+            if (_appliedCommand.HasOption("source"))
+            {
+                CheckForOutdated("--source");
+                //Forward source as multiple flags
+                foreach (var source in _appliedCommand.AppliedOptions["source"].Arguments)
+                {
+                    args.Add("--source");
+                    args.Add(source);
+                }
+            }
+
+            return args.ToArray();
+        }
+
+        /// <summary>
+        /// A check for the outdated specific options. If --outdated not present,
+        /// these options must not be used, so error is thrown
+        /// </summary>
+        /// <param name="option"></param>
+        private void CheckForOutdated(string option)
+        {
+            if (!_appliedCommand.HasOption("outdated"))
+            {
+                throw new Exception(string.Format(LocalizableStrings.OutdatedOptionOnly, option));
```

Comment:
We should throw `GracefulException` for any errors that are caused by the user, such as a conflicting options.